models/stg_apple_search_ads__ad_group_history.sql

```diff
@@ -20,26 +19,23 @@ fields as (
 ),
 
 final as (
-    
     select 
-        _fivetran_synced,
         modification_time as modified_at,
-        row_number() over (partition by id order by modification_time desc) = 1 as is_most_recent_record,
-        id as ad_group_id,
+        organization_id,
         campaign_id,
+        name as ad_group_name,
+        id as ad_group_id,
+        status as ad_group_status,
         start_time as start_at,
         end_time as end_at,
-        name as ad_group_name,
-        organization_id,
-        serving_state_reasons,
-        serving_status,
-        status
+        row_number() over (partition by id order by modification_time desc) = 1 as is_most_recent_record
     from fields
     {% if target.type == 'snowflake' -%}
-        where deleted = 'false'
+    where deleted = 'false'
```

Comment:
Is this needed? I don't believe I have encountered an issue before where `not deleted` didn't work on Snowflake? I wonder if "where deleted is false" doesn't work, but I think "not deleted" should be fine.

Reply:
Now that I think about it... Do we want to remove deleted ads? I would assume we still want to see the cost and values associated with those ads even though they were deleted. What are your thoughts?

Reply:
This field looks like it is only present in history models. So I believe it's considered a safe delete or something of that sort so it shouldn't affect the actual results of the report. However, to be safe, we _could_ just rely on is_most_recent_record, assuming that that record is not a deleted history record 🤔

Reply:
Yeah I say we rely on the most recent record field as this is the nature we have applied to our other Ad packages in this update.